torch/csrc/jit/passes/quantization.cpp

```diff
@@ -630,23 +643,41 @@ class InsertQuantDeQuantHelper {
   std::tuple<c10::QScheme, QParamMap> getQSchemeAndQParamMap(
       script::Module& module,
       Node* n);
+  void checkQScheme(Graph* g, c10::QScheme qscheme) {
+    if (qscheme_for_graph_.count(g)) {
+      TORCH_CHECK(toAffine(qscheme_for_graph_.at(g)) == toAffine(qscheme) ||
+
+                  "Quantizing same graph with different types of "
+                  "QSchemes is not supported.\n",
+                  " Expecting:",
+                  c10::toString(qscheme_for_graph_.at(g)),
+                  " Got:",
+                  c10::toString(qscheme));
+    } else {
+      qscheme_for_graph_[g] = qscheme;
+    }
+  }
   c10::optional<script::Module> findChildModuleToQuantize(
       script::Module& module,
       Value* child_instance);
   void collectObserverNodesAndValueToQuantize(script::Module& module, Value*);
-  void removeObservers(script::Module& module);
-  void removeObservers(script::Module& module, Graph* g);
+  void cleanup(script::Module& module);
```

Comment:
If you decide to use an abstract name `cleanup`, please add comments describing what exactly this cleanup does.